process_jartic_simple_v2.py

```
@@ -0,0 +1,205 @@
+                                        if '2024/01' in time_str:  # Quick filter for January 2024
+                                            total_count += 1
+                                    except:
+                                        continue
+                except:
+                    continue
+    
+    print(f"Total measurements to process: {total_count:,}")
+    return total_count
+
+
+def process_jartic_archive_with_total(archive_path: Path, output_path: Path, start_date: datetime, 
```

Ruff (0.12.2)

`process_jartic_archive_with_total` is too complex (12 > 10)

(C901)